proms-resources/clinical-data/src/main/resources/SLING-INF/content/Extensions/DashboardViews/PHQ9View.json

```
@@ -0,0 +1,8 @@
+{
+  "jcr:primaryType": "cards:Extension",
+  "cards:extensionPointId": "cards/coreUI/dashboardviews/entry/Cardio",
+  "cards:extensionName": "PHQ9 View",
+  "cards:extensionRenderURL": "asset:cards-dataentry.PromsView.js",
```

PromsView is in the `proms-homepage` module
```suggestion
  "cards:extensionRenderURL": "asset:proms-homepage.PromsView.js",
```